torch/csrc/jit/script/compiler.cpp

```diff
@@ -565,8 +565,14 @@ static std::shared_ptr<SugaredValue> tryEmitBuiltin(
     num_outputs = *value;
   }
 
-  for(size_t i = 0; i < num_outputs; ++i)
-    n->addOutput();
+  for(size_t i = 0; i < num_outputs; ++i) {
+    if (n->kind() == aten::chunk) {
+      // special case for chunk as the schema return only have one Tensor[] type
+      n->addOutput()->setType(DynamicType::get());
```

Comment:
The `->setType(DynamicType::get())` is redundant, right? (just checking, if so I think it's nice to have it here to differentiate it from the other case)

Reply:
Yes, it is redundant, it just makes the output type clear for special casing chunk.